server/repository/src/migrations/v2_02_00/item_ven.rs

```diff
@@ -0,0 +1,40 @@
+use crate::{migrations::sql, StorageConnection};
+
+pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
+    #[cfg(feature = "postgres")]
+    sql!(
+        connection,
+        r#"
+          CREATE TYPE ven_category AS ENUM (
+            'V',
+            'E',
+            'N',
+            'NOT_ASSIGNED'
+          );
+        "#
+    )?;
+
+    const VEN_CATEGORY_ENUM_TYPE: &str = if cfg!(feature = "postgres") {
+        "ven_category"
+    } else {
+        "TEXT"
+    };
+
+    sql!(
+        connection,
+        r#"
+            ALTER TABLE item ADD COLUMN strength TEXT;
```

Comment:
actually not certain that this is much used in OG - it was added fairly recently and afaik mostly the name is stuffed with this detail and not split out into the strength field 🤷

Reply:
Yeah, had some fun trying to find the field in the first place 😅 But figured if they wanted to populate it, it might as well come through? Unless you think this is causing more confusion than help, maybe we just don't support the strength column in the rnr rows?

Reply:
yeah, I thought the same.. might as well bring it through. They had strength on the R&R form, so it makes sense to store it somewhere